src/fiatExchanges/quotes/FiatConnectQuote.ts

```diff
@@ -152,12 +208,37 @@ export default class FiatConnectQuote extends NormalizedQuote {
     return this.quoteResponseKycSchema?.kycSchema
   }
 
-  // TODO: Dynamically generate time estimation strings
-  getTimeEstimation(): SettlementTime {
-    // payment method can only be bank or fc mobile money
-    return this.getPaymentMethod() === PaymentMethod.Bank
-      ? SettlementTime.ONE_TO_THREE_DAYS
-      : SettlementTime.LESS_THAN_24_HOURS
+  getTimeEstimation(): SettlementEstimation {
+    const fiatAccountInfo = this.quote.fiatAccount[this.getFiatAccountType()]
+
+    const lowerBound = fiatAccountInfo?.settlementTimeLowerBound
+      ? parseInt(fiatAccountInfo?.settlementTimeLowerBound)
+      : 0
+    const upperBound = fiatAccountInfo?.settlementTimeUpperBound
+      ? parseInt(fiatAccountInfo?.settlementTimeUpperBound)
+      : 0
+    if (
+      !fiatAccountInfo?.settlementTimeUpperBound ||
+      lowerBound < 0 ||
+      upperBound < 0 ||
+      lowerBound > upperBound
+    ) {
+      // payment method can only be bank or fc mobile money
+      return this.getPaymentMethod() === PaymentMethod.Bank
```

Comment:
This is more of a nit, but I get a little nervous when assumptions are made about what type a value would be in a way that won't be caught by the TS compiler if that value were able to take on more types; e.g., once `getPaymentMethod` can plausibly return payment types other than `Bank` and `FiatConnectMobileMoney`, it's very probable that this line wouldn't get updated to reflect that.

Kind of a pain to deal with since `PaymentMethod` encompasses payment types that could _never_ be on a FiatConnectQuote, and I'm not sure the best way to do this such that TS will complain if a value was added to the `PaymentMethod` enum and _wasn't_ explicitly dealt with here. For now maybe add a TODO: to ensure that this gets updated once more payment types are possible?

Reply:
one option is to define a default that is not payment method-specific, and default to that if the payment method is not bank or mobile money